front/src/Components/Menu/AboutRoomSubMenu.svelte

```diff
@@ -48,6 +53,9 @@
     <section class="container-overflow">
         <h3>{mapName}</h3>
         <p class="string-HTML">{mapDescription}</p>
+        {#if mapLink}
+            <p>&gt; <a href={mapLink}>link to this map</a> &lt;</p>
```

Comment:
Maybe the link should open a new tab:

```suggestion
            <p>&gt; <a href={mapLink} target="_blank">link to this map</a> &lt;</p>
```

Reply:
Yeah, already thought about that myself. Will change and rebase it. :-)

Reply:
Done. I also added `class="string-HTML"` to the surrounding `<p>`, because the other `<p>`s also have it.
And `"..."` around the actual URL in `href=`.